siliconcompiler/scheduler.py

```diff
@@ -140,7 +140,9 @@ def _deferstep(chip, step, index, status):
         if 'RUNNING' in jobout or \
            'PENDING' in jobout or \
            'CONFIGURING' in jobout or \
-           'COMPLETING' in jobout:
+           'COMPLETING' in jobout or \
+           'SIGNALING' in jobout or \
+           'STAGE_OUT' in jobout:
```

Comment:
is there a definitive list of these somewhere? if so, lets add a link to it and make sure all are accounted for.
Also the watchdog timer is gone, so we can remove that code too.